providers/nitro/nitro-enclave/src/main.rs

```diff
@@ -22,15 +22,19 @@ fn main() {
         .flatten()
         .unwrap_or(6050);
 
+    let log_level = env_args
+        .next()
+        .map(|x| {
+            if x.to_lowercase() == "--verbose" || x.to_lowercase() == "-v" {
+                "debug"
+            } else {
+                "info"
+            }
+        })
+        .unwrap_or_else(|| "info");
+    let filter_layer = EnvFilter::try_new(log_level).unwrap();
```

Comment:
is there a need to do this from `&str`?
Maybe just `Level::INFO` / `Level::DEBUG` can be returned directly?